### ISSUE TITLE
Switch to another module than File::Slurp

### ISSUE THREAD

**Issue by petdance** [petdance/bobby-tables#100]
File::Slurp will break under Perl 5.30.  Switch to something else, if not just writing my own read_file.

**Comment by dz-at-tc** [comment]
Hi @petdance , I wasn't sure whether eliminating a dependency entirely or clarity via retaining a dependency was the priority here, so I did it both ways.  Or perhaps there is a door number three you had in mind? :)

**Comment by dz-at-tc** [comment]
Hey @petdance , so none of the above, eh?

We could squirrel away the do/local/<> idiom ugliness like so...

```
sub read_file {
    return do {local(@ARGV, $/) = shift; <>};
}
```

... if that makes any difference?

**Comment by petdance** [comment]
> Hey @petdance , so none of the above, eh?

No, I just haven't had time to work on it yet.  Other things have taken precedence.  I appreciate the PR.

**Comment by dz-at-tc** [comment]
Ahhh, but of course (in my best Grey Poupon :)

Apologies for the line noise.
